package.json

```diff
@@ -20,9 +20,7 @@
     "sails": "./bin/sails.js"
   },
   "scripts": {
-    "test": "mocha -b",
-    "prepublish": "npm prune",
-    "preinstall": "node ./lib/preinstall_npmcheck.js"
+    "test": "node node_modules/mocha/bin/mocha -b"
```

Comment:
There's no need to prepend `node node_modules/mocha/bin/mocha`, since all binaries are already in the path when running from NPM.
Line should be changed back to its previous code: ` "test": "mocha -b"`